app/data/data-contract/data-contract-public/src/main/kotlin/com/hedvig/android/data/contract/ContractGroup.kt

```diff
@@ -1,5 +1,8 @@
 package com.hedvig.android.data.contract
 
+import kotlinx.serialization.Serializable
+
+@Serializable
 enum class ContractGroup {
```

Comment:
I thought that enums are just serializable by default without having to annotate them with the annotation.
Did this break before you added the annotation?
Did it perhaps break when the `alias(libs.plugins.serialization)` serialization gradle plugin was not applied, but then it starts working when you add it, without even having to add the annotation?